nodepool/elements/contrail-builder/package-installs.yaml

```diff
@@ -1,13 +1,15 @@
+kernel_sources:
+packaging_tools:
```

Comment:
I'd prefer to keep using real package names, like "linux-headers-generic" and map them correctly in pkg-map. Your thoughts? :)

Reply:
I've changed it because it has a whole different name on ubuntu and centos so I wanted to "abstract" it away and define for each OS.

Reply:
I don't have any strong opinion on that, just thought that it seems more readable than using `
`devscripts` on centOS. Can change it back if requested ;-)

Reply:
Yeah, I would prefer to keep actual package names in package-installs.yaml